spec/services/mixtape_maker_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe MixtapeMaker do
+  describe '.generate' do
+    it 'makes a correctly formatted mixtape' do
+      artists = create_list(:artist, 3)
+      songs = create_list(:song, 4, artist: artists[0])
+      mixtape = MixtapeMaker.generate(2, artist_ids: artists.collect{ |obj| obj.id})
+      expect(mixtape).to be
+    end
+
+    it 'makes an empty mixtape' do
+      artist = create(:artist)
+
+      mixtape = MixtapeMaker.generate(0, artist_ids: artist)
+      expect(mixtape).to be_empty
+    end
+
+    it 'makes a mixtape without an artist' do
+      songs = create_list(:song, 6)
+
+      mixtape = MixtapeMaker.generate(artist_ids: nil)
+      expect(mixtape).to be
```

Comment:
What's the expectation testing here?

Seems like it is using the `number_of_songs` of `5` so ... maybe expect that the mixtape has 5 default songs?

Otherwise just testing that the mixtape isn't nil, isn't that satisfying a test.